Create mysql_version_eol_57.md (#60)
percona-platform/portal-doc: docs/advisors/checks/mysql_version_eol_57.md

# Percona Server and MySQL 5.7 End Of Life
## Description

> Percona Server and MySQL server 5.7 are declared End Of Life versions. The deadline is on  Oct 21, 2023. 

Reviewer: I would rephrase this paragraph to:

Percona Server for MySQL Server version 5.7 will reach End-of-life on October 21st, 2023.
Percona does not provide support or issue hotfixes for retired releases.

Author: We'll improve this later; as for now, WIP on final messages about 5.7 EOL and our offers on it.

> Percona Server and MySQL server 5.7 are declared End Of Life versions. The deadline is on  Oct 21, 2023. 

Reviewer: Did you mean Percona Server for MySQL Server 5.7?

Author: Both of them

After that date the software will be out of support and users that will still be running that version will not
get any further update or bug fix.

## Resolution

> Contact Percona to get help to plan the software migration from version 5.7 to 8.0, or to discover Percona extended support for version 5.7.

Reviewer: ```suggestion
[Contact Percona](https://www.percona.com/about/contact) to get extended support for version 5.7, or help you plan the migration to version 8.0 (latest).  
```

https://www.percona.com/about/contact

> https://www.percona.com/services/policies/percona-software-support-lifecycle

Reviewer: ```suggestion
For more information, check out the [Percona software support lifecycle information](https://www.percona.com/services/policies/percona-software-support-lifecycle)
```
